scalafmt-core/shared/src/main/scala/org/scalafmt/util/GitOps.scala

```diff
@@ -46,6 +46,7 @@ class GitOpsImpl(private[util] val workingDirectory: AbsoluteFile)
         Seq(
           "git",
           "ls-files",
+          "--full-name",
```

Comment:
What should the behavior be for files not in the directory? As I understand this, this will get ALL files, even ones not in the directory you are in.

Reply:
Can you double check that ?

That's not the behaviour that I'm seeing:

1. Using git from the commandline
2. In the refined test I just added

https://github.com/lloydmeta/scalafmt/blob/a5d46c9c6899391f0eab1a310660e73fc8888335/scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala#L96-L105

3. In the man pages

```
--full-name
           When run from a subdirectory, the command usually outputs paths
           relative to the current directory. This option forces paths to be
           output relative to the project top directory.
```

Reply:
Well u added a test to prove that that was the case, that's enough to shut me up :)